package.json

```diff
@@ -38,6 +38,7 @@
     "parse-server-push-adapter": "1.0.4",
     "parse-server-s3-adapter": "1.0.4",
     "parse-server-simple-mailgun-adapter": "1.0.0",
+    "parse-aws-firehose-logger-adapter": "acinader/parse-aws-firehose-logger-adapter",
```

Comment:
The loader should let you load from command line bu passing the module name, we designed it for that. You could even have a logger.js that export your configured adapter and pass ./logger.js as the adapter